content/rc/security/single-sign-on.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: Single sign-on
+linkTitle: Single sign-on
+description: Single sign-on (SSO) integration with Redis Cloud
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+---
+
+As an alternative to the traditional email/password account authentication method, you can sign up for a Redis Cloud account and sign into the [admin console](https://app.redislabs.com/new/) with [single sign-on (SSO)](https://en.wikipedia.org/wiki/Single_sign-on). This allows you to use one set of credentials, managed by your [identity provider](https://en.wikipedia.org/wiki/Identity_provider), to access multiple websites.
+
+## Social login
+
+One SSO option that Redis Cloud offers is [social login](https://en.wikipedia.org/wiki/Social_login), which allows you to use an existing social media account to create or sign into a Redis Cloud account.
+
+Redis Cloud supports the following social logins:
```

Comment:
maybe add links?? not sure if it's worth it, just a thought

Reply:
Added links.